easybuild/easyblocks/a/aocc.py

```diff
@@ -67,6 +67,10 @@ def __init__(self, *args, **kwargs):
 
         self.clangversion = self.cfg['clangversion']
 
+        # Starting from LLVM v16, the folder naming is based on the major version of clang.
+        if LooseVersion(self.clangversion) >= LooseVersion('16.0.0'):
+            self.clangversion = '16'
+
     def _aocc_guess_clang_version(self):
```

Comment:
We should update the list with known AOCC versions, since this simplifies writing the respective EasyConfigs